homeassistant/components/withings/sensor.py

```diff
@@ -0,0 +1,639 @@
+"""Sensors flow for Withings."""
+import typing as types
+
+import nokia
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.util import slugify
+
+from . import const
+from .common import (
+    _LOGGER,
+    NotAuthenticatedError,
+    WithingsDataManager,
+)
+
+# There's only 3 calls (per profile) made to the withings api every 5
+# minutes (see throttle values). This component wouldn't benefit
+# much from parallel updates.
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+        hass: HomeAssistantType,
+        entry: ConfigEntry,
+        async_add_entities: types.Callable[[types.List[Entity], bool], None]
+):
+    """Set up the sensor config entry."""
+    profile = entry.data[const.PROFILE]
+    credentials = nokia.NokiaCredentials()
+    credentials.__dict__.update(entry.data[const.CREDENTIALS])
+
+    def credentials_saver(credentials_param):
+        _LOGGER.debug("Saving updated credentials.")
+        entry.data[const.CREDENTIALS] = credentials_param
+        hass.config_entries.async_update_entry(entry, data={**entry.data})
+
+    _LOGGER.debug(
+        "Creating nokia api instance with credentials %s",
+        credentials
+    )
+    api = nokia.NokiaApi(
+        credentials,
+        refresh_cb=(lambda token: credentials_saver(
+            api.credentials
+        ))
+    )
+
+    _LOGGER.debug(
+        "Creating withings data manager for profile: %s",
+        profile
+    )
+    data_manager = WithingsDataManager(
+        profile,
+        api
+    )
+
+    _LOGGER.debug("Confirming we're authenticated")
+    try:
+        data_manager.check_authenticated()
+    except NotAuthenticatedError:
+        # Trigger new config flow.
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                const.DOMAIN,
+                context={
+                    'source': const.SOURCE_USER,
+                    const.PROFILE: profile
+                },
+                data={}
+            )
+        )
+        return
+
+    entities = create_sensor_entities(hass, data_manager)
+    async_add_entities(entities, True)
+
+
+def create_sensor_entities(
+        hass: HomeAssistantType,
+        data_manager: WithingsDataManager
+):
+    """Create sensor entities."""
+    entities = []
+
+    measures = hass.data.get(const.DOMAIN, {}) \
+        .get(const.CONFIG, {}) \
+        .get(const.MEASURES, list(WITHINGS_MEASUREMENTS_MAP))
+
+    for attribute in WITHINGS_ATTRIBUTES:
+        if attribute.measurement not in measures:
+            _LOGGER.debug("Skipping measurement %s as it is not in the list of measurements to use", attribute.measurement)  # pylint: disable=line-too-long  # noqa: E501
```

Comment:
Instead of disabling the line too long, just make the line not so long. Python will automatically concatenate multiple strings if there is just whitespace in between.

```python
_LOGGER.debug(
    "Skipping measurement %s "
    "as it is not in the list of measurements", attribute.measurement
)
```